dispatch/modules/content/managers.py

```diff
@@ -9,4 +9,22 @@ def get(self, *args, **kwargs):
             kwargs['parent'] = kwargs['pk']
             kwargs['head'] = True
             del kwargs['pk']
-        return super(PublishableManager, self).get(*args, **kwargs)
+
+        """If the url requested includes the querystring parameters 'version' and 'preview_id',
+        get the article with the specified version and preview_id.
+
+        Otherwise, get the published version of the article.
+        """
+
+        if len(args) > 0:
```

Comment:
I think the build is failing because `request` is now a required parameter for `Article.objects.get`, so all existing calls to this method will fail.

To fix this, you can make `request` an optional keyword argument and call the method like this:

```python
article = Article.objects.get(request=request, slug=slug, is_published=True)
```

This _should_ avoid breaking changes to existing calls to `Article.objects.get`.